README.md

```diff
@@ -6,6 +6,7 @@
     1. [Toolchain](building/toolchain.md)
     2. [Building script](building/script.md)
     3. [Reference project repository](building/project.md)
+    4. [Setting up Windows](building/Windows.md)
```

Comment:
I would consider integrating this into` phoenix-rtos-doc/building/README.md` rather than creating a separate subsection. Unless this was previously decided differently, in which case I would also provide a reference in the mentioned` .md` file, perhaps under a section titled "Host Operating System", linking to this file.